### ISSUE TITLE
[BUG] Get-PnPFlow Times Out after 100 Seconds

### ISSUE THREAD

**Issue by cprovpo** [pnp/powershell#3794]
### Reporting an Issue or Missing Feature
Get-PnPFlow cmdlet times out after 100 seconds.

### Expected behavior 
The command should return a list of Power Automate flows from the tenant.

### Actual behavior 
Issue occurred on multiple environments, with and without the -AsAdmin switch passed.
![image](https://github.com/pnp/powershell/assets/139259660/433e6909-5884-4abb-bc61-914d2e313e9a)

### Steps to reproduce behavior
Connect-PnPOnline -Url $spAdminUrl -Interactive
Get-PnPFlow
Get-PnPFlow -AsAdmin

### What is the version of the Cmdlet module you are running?
2.3.69 (updated after experiencing [this bug](https://github.com/pnp/powershell/issues/3640)

### Which operating system/environment are you running PnP PowerShell on?
- [ X ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by cprovpo** [comment]
It appears that this was previously raised as a discussion here: https://github.com/pnp/powershell/discussions/1871

**Comment by pkbullock** [comment]
I too am seeing the same error when when running Get-PnPFlow in a windows terminal, PS7 prompt, version 2.3.66

**Comment by jackpoz** [comment]
I opened a PR at https://github.com/pnp/powershell/pull/3820 to fix the issue, if you want to give it a try.

**Comment by gautamdsheth** [comment]
Should be fixed with tomorrow’s nightly builds .

**Comment by renewtx** [comment]
Hello -- I was able to install the pre-release locally (on my machine) and it did appear to fix the issue -- Thank You!  However, I have a  question (a couple actually):

1. Any idea when the next full release will be available that will fix this?
2. I was hoping to use this via an Azure Runbook -- do  you know if there is a way to install/deploy these pre-release versions for use in Azure Runbooks?

Thanks

**Comment by renewtx** [comment]
For what it's worth, I figured it out (maybe this will help someone else in a similar situation).  I installed the needed (updated) module locally on my laptop under PowerShell v7.  Once installed, I found the module locally under C:\Users\UserName\Documents\PowerShell\Modules\PnP.PowerShell.  I then zipped that folder using the same file name as the module name.  Then I removed the previous version of the module from my Azure Runbook modules and replaced it by importing the ZIP file I created as the replacement for PnP.PowerShell.   Now I am able to run my runbook without encountering the 100s HTTP timeout error.
